src/index.html

```diff
@@ -1,13 +1,290 @@
+                    src="./images/categories/audio-wide.svg"
+                    alt="Audio">
+                  </a>
+                </div>
+                <h3 class="category__title">Audio</h3>
+              </article>
+
+              <article class="category">
+                <div class="category__photos">
+                  <a href="#" class="category__link category__link--wide">
+                    <img class="category__photo"
+                    src="./images/categories/accessories-wide.svg"
+                    alt="Accessories">
+                  </a>
+
+                  <a href="#" class="category__link category__link--square">
+                    <img class="category__photo"
+                    src="./images/categories/accessories-sq.svg"
+                    alt="Accessories">
+                  </a>
+                </div>
+                <h3 class="category__title">Accessories</h3>
+              </article>
+
+              <div class="category__button">
+                <a class="category__button-view-all"
+                href="#"
+                >
+                View all
+                </a>
+              </div>
+            </div>
+          </section>
+        </div>
+      </div>
+    </main>
+
+    <footer class="footer">
+      <div class="dividing__picture"></div>
+
+      <div class="container">
+        <section class="about-us">
+          <h2 class="
+          about-us__title
+          section-title
+          section-title--align--left">
+            About us
+          </h2>
+          <div class="about-us__description">
+            <p>We're a London-based tech company on a mission to remove barriers between people and technology.</p>
+            <p>To do this, we're crafting intuitive, flawlessly connected products that improve our lives without getting in the way.
+              No confusing tech-speak. No silly product names. Just artistry, passion and trust.
+              And products we're proud to share with our friends and family. Simple.
+            </p>
+            <p>Because beautiful tech shouldn't be complicated.</p>
+          </div>
+        </section>
+      </div>
+
+      <div class="container">
+        <section class="contact-us">
+          <h2 class="section-title section-title--align--left">
+            Contact us
+          </h2>
+
+          <div class="contact-us__content">
+            <form class="form contact-us__form"
+            action="#"
+            method="post"
+            onsubmit="this.reset(); return false;"
+            >
+              <input
+              type="text"
+              class="form__field"
+              name="name"
+              placeholder="Name"
+              required
+              >
+
+              <input
+              type="text"
+              class="form__field"
+              name="email"
+              placeholder="Email"
+              required
+              >
+
+              <textarea
+              type="textarea"
```

Comment:
The attribute type is not allowed on the element textarea accordingly to the living standard